### DISCUSSION TITLE
about adding bills to existing customer

### DISCUSSION THREAD

**Discussion by dtbj** [saleor/saleor#8887] [Q&A] [answered]
When customer orders the bill is on the customer's name right? and listed in my orders so what I want is like some of the customers are hybrid users like they purchase goods offline as well as sometimes online so like if they are purchasing offline so is there any way admin can bill into their account directly?

**Comment by karolkielecki** [comment, marked as answer]
@dtbj Hi! Yes, each order has to be assigned to a customer. Store managers can create orders/draft orders and assign customers to them in the dashboard. Also, we're working on the click and collect feature--it will be released with 3.1. Maybe it will be helpful in your use case: https://github.com/saleor/saleor/projects/12.

**Reply by dtbj** [reply]
Thank you so much for replying, it's perfect solution for me.
